docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,18 @@
 
 //=== Regressions
 
+== {es} version 7.2.1
+
+=== Bug Fixes
+
+* Don't persist model state at the end of lookback if the lookback did not
+generate any input {ml-pull}527[#527] (issue: {ml-issue}519[#519])
+* Don't write model size stats when job is closed without any input
+{ml-pull}516[#512] (issue: {ml-issue}394[#394])
```

Comment:
516 and 512 are inconsistent.  512 is the master branch PR for this change and 516 is the 7.2 branch backport PR.  I'm not sure which one is standard to link to but I guess whichever is preferred it should be consistent for the link and text.

Reply:
Good catch, thanks!